app/views/MainTemplate.scala.html

```diff
@@ -1,54 +1,98 @@
-  feedbackOnSignOut: Boolean = true)(mainContent: Html)(implicit request: Request[_], messages: Messages, appConfig: AppConfig)
-
-@serviceInfoContent = {}
-
-@sidebarContent = {
-    @if(sidebarLinks.isDefined) {
-        @sidebar(sidebarLinks.get, Some("sidebar"))
-    }
+* Copyright 2021 HM Revenue & Customs
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*@
+
+@import uk.gov.hmrc.hmrcfrontend.views.config.StandardBetaBanner
+
+@this(govukLayout: GovukLayout,
+      standardBetaBanner: StandardBetaBanner,
+      hmrcReportTechnicalIssue: HmrcReportTechnicalIssue,
+      hmrcLanguageSelect: HmrcLanguageSelect,
+      hmrcTimeoutDialog: HmrcTimeoutDialog,
+      hmrcStandardHeader: HmrcStandardHeader,
+      hmrcStandardFooter: HmrcStandardFooter,
+      hmrcTrackingConsentSnippet: HmrcTrackingConsentSnippet)
+
+@(pageTitle: String,
+ serviceName: Option[String] = None,
+ showSignOut: Boolean = true,
+ feedbackOnSignOut: Boolean = true,
+ backLinkContent: Option[Html] = None
+)(mainContent: Html)(implicit request : Request[_], messages: Messages, appConfig: config.AppConfig)
+
+@navTitle = @{
+messages("common.agentService", pageTitle, messages("common.agentService"))
 }
 
-@govukWrapper(appConfig = appConfig,
-               title = title,
-               backLinkContent = backLinkContent,
-               mainClass = mainClass,
-               bodyClasses = bodyClasses,
-               sidebar = sidebarContent,
-               contentHeader = contentHeader,
-               mainBody = article(mainContent),
-               serviceInfoContent = serviceInfoContent,
-               scriptElem = scriptElem,
-               showSignOut = showSignOut,
-               feedbackOnSignOut = feedbackOnSignOut
+@scripts = {
+<script src='@routes.Assets.at("lib/govuk-frontend/govuk/all.js")'></script>
+<script src='@routes.Assets.at("lib/hmrc-frontend/hmrc/all.js")'></script>
+<script>window.GOVUKFrontend.initAll();</script>
+<script>window.HMRCFrontend.initAll();</script>
+}
+
+@beforeContentHtml = {
+@hmrcLanguageSelect(LanguageSelect(
+language = if(messages.lang.language == "en") En else Cy,
+languageLinks =
+En -> appConfig.routeToSwitchLanguage("english").url,
+Cy -> appConfig.routeToSwitchLanguage("cymraeg").url
+))
+@backLinkContent
+}
+
+@head = {
+@hmrcTrackingConsentSnippet()
+
+<link rel="stylesheet" type="text/css" href='@routes.Assets.at("stylesheets/application.css")' media="all">
+
+@hmrcTimeoutDialog(TimeoutDialog(
```

Comment:
I believe the the timeout dialog should still be wrapped inside an if(showSignOut), at least that's both what we had inside the GovukWrapper as well as what we still have in other services